zapier/introduction/what-is-zapier/zapier-common-terminology.md

```diff
@@ -0,0 +1,57 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Zapier Dependencies
+
+---
+## Content
+
+Now that you have your account, you are ready to start using Zapier.
+
+The first step is getting used to some of the terms that you will see. Here is a list to help you get started[1]:
+- **zap**
+  - this refers to the automated workflow that connects apps and services
+  - think of a zap as your entire integration
+  - e.g. we have previously mentioned that you can get email notification for each Google form completion. This is a zap.
+- **trigger**
+  - this represents an event that starts a zap
+  - in our example, the trigger would be the submittal of the Google form
+- **action**
+  - this represents an event that will be performed by our zap after a trigger
+  - going back to the example, the action is getting the email notification
+- **task**
+  - this is more a general term that refers to any single action performed by Zapier
+  - in our example, submitting a form **or** receiving an email notification are both actions
+  - as you have already noticed, both triggers and tasks can be actions
```

Comment:
> - as you have already noticed, both triggers and tasks can be actions

but you say that 

>  - this represents an event that will be performed by our zap after a trigger

it's contradictory cause action is defined as being after a trigger, but then you say that a trigger can be an action


also maybe mention that a zap is made out of tasks basically